### ISSUE TITLE
[guide] openid azure how-to

### ISSUE THREAD

**Issue by jamiltz** [couchbaselabs/couchbase-mobile-portal#449]
See https://github.com/couchbase/couchbase-lite-ios/issues/1392

**Comment by jamiltz** [comment]
Notes from @sethrosetter 

>Good Resource - 
    https://azure.microsoft.com/en-us/documentation/articles/app-service-mobile-how-to-configure-active-directory-authentication/

> Create a new app

> Use https://sts.windows.net/{tenant-id}

> Can be found by looking at app endpoints in the bottom bar of the AD application you create. 

> https://login.microsoftonline.com/{tenant_id}/federationmetadata/2007-06/federationmetadata.xml

> In azure portal:
> Copy:
    - Client id == "client_id"
    - Key  == "validation_key"

> Set Reply Url to the "callback_url". Example: http://localhost:4984/default/_oidc_callback
> Set App ID Uri - https://Couchbase/sync-gateway-oidc-test
> https://<tenant-name>/<app-name>


 > {
   "interface":":4984",
   "log":["*"],
   "databases": {
      "default": {
        "server": "http://localhost:8091",
        "bucket": "default",
        "oidc": {
          "default_provider":"azure",
          "providers": {
            "azure": {
                "issuer":"https://sts.windows.net/{tenant_id}/",
                "client_id":"abc",
                "validation_key":"xyz",
                "callback_url":"http://localhost:4984/default/_oidc_callback",
                "register":true
            }
          }
        }
      }
   }
}

**Comment by jamiltz** [comment]
@sethrosetter Can you explain the process for testing with Azure as the OP? The [current docs](http://developer.couchbase.com/documentation/mobile/1.3/develop/guides/authentication/openid/google/index.html#setting-up-couchbase-lite) are currently based on Grocery Sync with Google login. If there's a version of it for Azure we could use it in the docs as well similarly to Google login.

**Comment by jamiltz** [comment]
For the 1.3 release, I paired with @adamcfraser [to document the way to run the OIDC flow in the browser](http://developer.couchbase.com/documentation/mobile/1.3/develop/guides/authentication/openid/google/index.html#setting-up-sync-gateway) without CBL. It'd be nice to have that for Azure in 1.3.1. Are there any differences you've encountered that I should have in mind before starting to write the same for Azure?
